.github/workflows/CI.yml

```diff
@@ -31,25 +31,26 @@ jobs:
           - os: windows-latest
             arch: x86
         include:
-          # Add a 1.3 job because that's what Invenia actually uses
+          # Add a 1.5 job because that's what Invenia actually uses
           - os: ubuntu-latest
-            version: 1.3
+            version: 1.5
```

Comment:
@omus was placing these in double quotes, not sure if we have a preferred version. But I think just picking one and being consistent across all CI files is best.